apps/2023/src/icons/link.jsx

```diff
@@ -0,0 +1,17 @@
+export default function Link({ width = 25, height = 28, color = 'white', className }) {
+  return (
+    <svg
+      width={width}
```

Comment:
Shall we provide the width and height with tailwind classes and let a user override those classes ? It'll allow for better control when working with breakpoints

Reply:
Done